### DISCUSSION TITLE
Bottom Sheet Component Guide disables mouse clicks

### DISCUSSION THREAD

**Discussion by karnendra93** [react-native-elements/react-native-elements#3596] [Q&A]
Hi I am new to RNElements. I need to use the BottomSheet component in my RN App. But when I go to the Bottom Sheet webpage, a transparent modal like thing slides from the bottom-up and disables mouse clicks, and I can't do anything. This doesn't happen on other components web pages. Of course, when I switch to older version guide, this doesn't happen. But I don't want to use older versions. 
Any help would be appreciated.
Thanks in advance

**Comment by arpitBhalla** [comment]
Fixed
